airflow/providers/cncf/kubernetes/backcompat/pod_runtime_info_env.py

```diff
@@ -14,13 +14,25 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""Classes for interacting with Kubernetes API"""
+"""
+Classes for interacting with Kubernetes API.
+
+This module is deprecated. Please use :mod:`kubernetes.client.models.V1EnvVar`.
+"""
+
+import warnings
 
 import kubernetes.client.models as k8s
 
+warnings.warn(
+    "This module is deprecated. Please use `kubernetes.client.models.V1EnvVar`.",
+    DeprecationWarning,
+    stacklevel=2,
```

Comment:
```suggestion
    stacklevel=3,
```

See the general comment